core/app/modules/import-export/assets/js/pages/import/import-content/components/import-content-footer/import-content-footer.js

```diff
@@ -26,14 +28,22 @@ export default function ImportContentFooter( { hasPlugins, hasConflicts, isImpor
 					} else {
 						onResetProcess();
 					}
+					if ( 'kit-library' === referrer && goBack ) {
+						goBack();
+					}
 				} }
 			/>
 
 			<Button
 				variant="contained"
 				text={ __( 'Import', 'elementor' ) }
 				color={ isImportAllowed ? 'primary' : 'disabled' }
-				onClick={ () => isImportAllowed && navigate( getNextPageUrl() ) }
+				onClick={ () => {
+					if ( 'kit-library' === referrer && approveImport ) {
+						approveImport();
```

Comment:
use a better name, maybe something like this:

```suggestion
						onImportClick();
```